resources/slack-markdown.template

```diff
@@ -6,6 +6,7 @@
 <% testsUrl = String.format("<%stests|here>", jobUrl)%>
 <% changesMessage = (changes?.url && changes?.msg) ? "${commitUrl} (by `${changes?.author?.id}`)" : "No push event to branch ${build?.pipeline}" %>
 <% stepsMessage = (steps?.size()!= 0) ? "`${steps?.size()}` (click ${artifactsUrl} and open `build.md` for further details)" : 0%>
+<% if (header != "") out.print "${header}" %>
```

Comment:
I was not able to reproduce same behavior as in below variables without directly using `out` response

Reply:
I'd say to use ?.trim() just in case the header is `null`

```suggestion
<% if (header?.trim()) out.print "${header}" %>
```

You could use, the below instead `out`

```
<%if(header?.trim()) {%>${header}<%}%>
```

Reply:
Used JSP style here: https://github.com/elastic/apm-pipeline-library/pull/784/commits/270d26eb17d2898ae7a4aff2eac863b92e18a7b4